DAQ_System/Source/main.cpp

```diff
@@ -40,6 +40,15 @@
 #define BUFFER_SIZE 21 // (4 linpot integers) * (4 bytes allocated per integer) = 16, 4 commas = 4 chars = 4 bytes, 1 more char & byte for the newline, 16 + 4 + 1 = 21
 using namespace application;
```

Comment:
Remove this line. I would like calling on a layer's code to be visually explicit